Formula/mapserver.rb

```diff
@@ -45,44 +45,39 @@ class Mapserver < Formula
   fails_with gcc: "5"
 
   def install
-    ENV.cxx11
-
-    args = %W[
-      -DWITH_CLIENT_WFS=ON
-      -DWITH_CLIENT_WMS=ON
-      -DWITH_CURL=ON
-      -DWITH_FCGI=ON
-      -DWITH_FRIBIDI=OFF
-      -DWITH_GDAL=ON
-      -DWITH_GEOS=ON
-      -DWITH_HARFBUZZ=OFF
-      -DWITH_KML=ON
-      -DWITH_OGR=ON
-      -DWITH_POSTGIS=ON
-      -DWITH_PYTHON=ON
-      -DWITH_SOS=ON
-      -DWITH_WFS=ON
-      -DPYTHON_EXECUTABLE=#{Formula["python@3.9"].opt_bin/"python3"}
-      -DPHP_EXTENSION_DIR=#{lib}/php/extensions
-      -DCMAKE_INSTALL_RPATH=#{rpath}
-    ]
+    python = which("python3.9")
 
     # Install within our sandbox
-    inreplace "mapscript/python/CMakeLists.txt" do |s|
-      s.gsub! "${PYTHON_LIBRARIES}", "-Wl,-undefined,dynamic_lookup"
-    end
+    inreplace "mapscript/python/CMakeLists.txt", "${PYTHON_LIBRARIES}", "-Wl,-undefined,dynamic_lookup" if OS.mac?
```

Comment:
We could probably pass `-DPYTHON_LIBRARIES=-Wl,-undefined,dynamic_lookup` to `cmake` instead of doing an `inreplace` here.

Reply:
I was a bit curious if this was still necessary. It was added back when using system Python 2 https://github.com/Homebrew/homebrew-core/commit/2909f1c848734c08cc4d12847b96f7dc80cde4ac. Not sure if it was trying to reduce linkage somewhere.

---

Anyway, feel free to change things outside of #107250. I'll do a rebase and fix any merge conflicts later today once a long timeout slot frees up.

Reply:
Might still be necessary to avoid `Python.framework` linkage in `_mapscript.so`.

This was important in the past to make sure that users can use Python libraries/bindings that include C extension modules with a non-Homebrew Python.

We don't really support this anymore, but it doesn't really cost much to keep doing. It's might also be important for any C extension modules built using macOS-provided Python (in case it links with a `Python.framework` in `Xcode.app` instead of the CLT), but I don't think we build any of those.